File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/CPP/codes/p34.cpp
//No args..

#include<iostream>
class Demo{
	public:
		Demo(){
			std::cout<< "In Constructor" <<std::endl;
			std::cout<< this <<std::endl;
		}
};
int main(){
	Demo obj;
	std::cout<< &obj <<std::endl;
}

File: java-main/java-main/NestedforloopAssi/p34.java
/*
1A 2B 3C
1A 2B 3C
1A 2B 3C */

class Demo{
	public static void main(String[] args){

		for(int i=1;i<=3;i++){

			int num=1;
			char ch='A';

			for(int j=1;j<=3;j++){

				System.out.print(num++);
				System.out.print(ch++ +" ");
			}
			System.out.println();
		}
	}
}
